src/stores.js

```diff
@@ -28,13 +28,63 @@ export const currentSection = writable({
 // about, swag, events, issues
 
 export const events = writable([
+	{
+		isOpened: false,
+		timeISO: '2023-09-27T16:00:00.000Z',
+		durationInMs: DurationMinute * 60,
+		name: 'IllaCloud LiveStream',
+		presenter: 'Wesley Cope',
+		//link: 'https://lu.ma/hf-kickoff-blr',
+		hasStarted: false,
+		sessions: []
```

Comment:
Last year we broke down the event into sessions. This is also important because a session is required to show some of the additional details for the event like the link.